### DISCUSSION TITLE
RuntimeError: mat1 and mat2 shapes cannot be multiplied (1x800 and 2x8)

### DISCUSSION THREAD

**Discussion by saai07** [mrdbourke/pytorch-deep-learning#1127] [Q&A]
in chapter 84 time stamp 13:14:23 , sir you have you used X-blob_test and the code is working fine but when i am using  ,there is error coming RuntimeError: mat1 and mat2 shapes cannot be multiplied (1x800 and 2x8) 

i did refer to the book there u have used train instead of test ?

sir pls explain me why i am getting this error and how to resolve it  ?

**Comment by saai07** [comment]
![Screenshot (39)](https://github.com/user-attachments/assets/883f0676-68a1-49de-8879-14adfeee401d)
![Screenshot (40)](https://github.com/user-attachments/assets/4c8e31dc-9f8d-4734-a7bd-2786a6e2b59f)
![Screenshot (42)](https://github.com/user-attachments/assets/20ad14eb-31b1-4b24-838f-38fb35356d3d)

![Screenshot (43)](https://github.com/user-attachments/assets/212c87dd-5eb8-4994-bd6e-1aaefa0ba3ae)


ss of my code

**Comment by PixelCoder210** [comment]
in training loop should be 
```
saiz(X_blob_train) # not saiz(X_blob_train).squeeze()
```
